### ISSUE TITLE
One of the monitors not respecting brightness settings

### ISSUE THREAD

**Issue by borovac** [xanderfrangos/twinkle-tray#6]
When I first installed the app, everything worked. Installation was from the MS store. 

In the mean time my main monitor (Lenovo) stoped reacting to changes of brightness. At the same time Dell changes are visible. Nothing happens even if I link the monitor values. 
Today I installed the latest version from Github, but nothing changed (except for Lenovo name now handled correctly - thanks for that). 
Both monitors are connected via DP, if that matters, to AMD RX580. Drivers are up to date, as per Radeon software. Difference is that Dell is pivoted to vertical position. Lenovo is horizontal.

What happens: 
I make changes to sliders. Dell reacts instantly. For Lenovo nothing happens.
I close the Twinkle tray windows and opet it again. Lenovo values are defaulted to 50.

**Comment by xanderfrangos** [comment]
It's possible that the DDC/CI option isn't enabled on the Dell monitor or it doesn't accept changes. I've also heard of monitors that won't update the brightness unless you turn the monitor off/on, which is unfortunate. 

Can you provide the model number for the Dell monitor?

**Comment by borovac** [comment]
It's Lenovo l24q-10, a cheap 1440p model. 
The weird thing is that everything worked after first install. The only thing I can think of are AMD and Windows updates.
I'll try it with HDMI cable sometime today or tomorrow and report back.

**Comment by borovac** [comment]
OK, so I tried HDMI and it didn't work. But then I noticed (when the sun came down) that even the night light from Windows is not working on Lenovo. 
So, I reseted the AMD control panel and overriden the settings for picture control.

Now everything works.

PS: AMD has a new control panel for the graphics card, so I guess during the update something happened and it circumvented the changes for the monitor.

![2020-01-21_19-11-04](https://user-images.githubusercontent.com/12745167/72830844-e5149300-3c81-11ea-80e2-3c4cac39ddcc.jpg)
